src/bundle/Resources/views/parts/navigation.html.twig

```diff
@@ -14,14 +14,23 @@
                     'ancestorClass': 'active'
                 }) }}
             </ul>
-            <ul class="nav navbar-nav ml-md-auto">
-                {# @todo use knp for user menu as well #}
-                <li class="nav-item">
-                    <a class="nav-link" href="{{ url('logout') }}">
-                        {{ 'authentication.logout'|trans|desc('Logout') }}
-                    </a>
-                </li>
-            </ul>
+            <div class="ez-user-menu ml-md-auto">
+                <div class='ez-user-menu__name-wrapper'>
+                    {% set user = admin_ui_config.user.user%}
+                    <span class="ez-user-menu__name">{{ user.name }}</span>
+                    {% if admin_ui_config.user.profile_picture_field.value.uri is not null %}
```

Comment:
Q: @webhdx what will happen if Field is null, not just uri property (you're returning it from provider in some edge case)?
(not a Twig expert).